src/components/GlobalHeader/index.tsx

```diff
@@ -3,20 +3,14 @@ import { Icon } from 'antd'
 import Link from 'umi/link'
 import Debounce from 'lodash-decorators/debounce'
 import RightContent from './RightContent'
+import { GlobalHeaderRightProps } from './RightContent'
 
 const styles = require('./index.less')
 
-interface GlobalHeaderProps {
-  onCollapse: Function
-  isMobile: boolean
-  collapsed: boolean
-  logo: string
-}
-
-export default function GlobalHeader (props: GlobalHeaderProps) {
+export default function GlobalHeader (props: GlobalHeaderRightProps) {
 
   const { onCollapse, isMobile, collapsed, logo } = props
-  @Debounce(600)
+  Debounce(600)
```

Comment:
Does this change work? I think it won't work - because it won't take the function below as an argument. 
Either apply it as a normal call, or try to understand why we need it - then we can remove it.

- [ ] redo Debounce refactoring

Reply:
The change compiled well. Undoing it now, since it needs looking into.